src/DependencyManagement/DependencySnapshotPurger.cs

```diff
@@ -128,34 +128,17 @@ private DateTime GetSnapshotAccessTimeUtc(string path, ILogger logger)
 
         private static TimeSpan GetHeartbeatPeriod()
         {
-            return GetEnvironmentVariableTimeSpanValue("MDHeartbeatPeriod") ?? TimeSpan.FromMinutes(60);
+            return PowerShellWorkerConfiguration.GetTimeSpan("MDHeartbeatPeriod") ?? TimeSpan.FromMinutes(60);
```

Comment:
Should we move all default values to constants?

Reply:
What I like about the current code is that the names, the types, and the default values are all on the same line, which makes it super-obvious and easy to maintain. I've seen code having constants for default values in similar cases, and I believe this is an anti-pattern: it introduces a level of indirection that makes values harder to look up, and creates an opportunity for copy-paste bugs, without providing valuable benefits.

Reply:
I do see your point. I am fine if you'd like to keep it as is.